src/main/java/online/talkandtravel/exception/data/FailedToReadJsonException.java

```diff
@@ -1,21 +1,21 @@
 package online.talkandtravel.exception.data;
 
-import online.talkandtravel.exception.model.ApiException;
+import online.talkandtravel.exception.model.HttpException;
 import org.springframework.http.HttpStatus;
 
 /**
  * Exception thrown when a JSON file fails to be parsed or read.
  *
  * <p>This exception is used to indicate that there was an error in processing a JSON file,
- * typically due to issues with the file format or content. It extends {@link ApiException} to
+ * typically due to issues with the file format or content. It extends {@link HttpException} to
  * integrate with the application's error handling mechanism, providing both a descriptive message
  * and an HTTP status code.
  *
  * <p>The exception message is formatted with the file path of the JSON file that could not be read.
  * The HTTP status is set to 409 (Conflict) to reflect that the request could not be completed due
  * to a conflict with the current state of the resource.
  */
-public class FailedToReadJsonException extends ApiException {
+public class FailedToReadJsonException extends HttpException {
```

Comment:
Your suggestion requires a separate ticket and additional discussion maybe on meeting. So, my suggestion is to create that ticket and discuss it on meeting, but this code leave as it is. Because the goal of this ticket was to create WebSocket exception handling mechanism, and it was created here.